launcher/minecraft/mod/ResourceFolderModel.cpp

```diff
@@ -415,7 +419,25 @@ QVariant ResourceFolderModel::data(const QModelIndex& index, int role) const
                     return {};
             }
         case Qt::ToolTipRole:
+            if (column == NAME_COLUMN) {
+                if (at(row).isSymLinkUnder(instDirPath())) {
+                    return m_resources[row]->internal_id() +
+                        tr("\nWarning: This resource is symbolicly linked from elsewhere. Editing it will also change the origonal") +
```

Comment:
.

Reply:
```suggestion
                        tr("\nWarning: This resource is symbolically linked from elsewhere. Editing it will also change the original.") +
```